crates/router/src/configs/settings.rs

```diff
@@ -130,6 +130,7 @@ pub struct Settings {
     pub frm: Frm,
     #[cfg(feature = "olap")]
     pub report_download_config: ReportConfig,
+    pub opensearch_config: OpensearchConfig,
```

Comment:
Can you populate the `development.toml`, `docker_compose.toml` & `config.example.toml`